cmd/jimmsrv/main.go

```diff
@@ -155,6 +155,17 @@ func start(ctx context.Context, s *service.Service) error {
 
 	logSQL, _ := strconv.ParseBool(os.Getenv("JIMM_LOG_SQL"))
 
+	crossModelQueryTimeout := 5 * time.Second
+	crossModelQueryTimeoutRaw := os.Getenv("JIMM_CROSS_MODEL_QUERY_TIMEOUT")
+	if crossModelQueryTimeoutRaw != "" {
+		expiry, err := time.ParseDuration(crossModelQueryTimeoutRaw)
+		if err != nil {
+			zapctx.Error(ctx, "failed to parse cross model query timeout duration", zap.Error(err))
```

Comment:
should we exit, too? i mean, this way JIMM will come up with unexpected configuration. If admin sets it, let them set it right - exit.